<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="dateLeftSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="datePrevEkadashi">
        <property name="text">
         <string>« P&amp;revious Ekādaśī</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dateLabel">
        <property name="text">
         <string>Search starting from date:</string>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <property name="buddy">
         <cstring>dateEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2020</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string extracomment="qwe">yyyy-MM-dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="timeSpec">
         <enum>Qt::LocalTime</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="todayButton">
        <property name="text">
         <string>Today</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dateNextEkadashi">
        <property name="text">
         <string>&amp;Next Ekādaśī »</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="dateRightSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalBigLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_brief">
         <attribute name="title">
          <string>Summary</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="dateLocationGroup">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>280</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Location</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="locationVLayout">
               <item>
                <layout class="QHBoxLayout" name="locationHLayout">
                 <item>
                  <widget class="QLabel" name="locationLabel">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>&amp;Location:</string>
                   </property>
                   <property name="margin">
                    <number>5</number>
                   </property>
                   <property name="buddy">
                    <cstring>locationComboBox</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="locationComboBox"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="latLongHLayout">
                 <item>
                  <widget class="QLabel" name="latitudeLabel">
                   <property name="toolTip">
                    <string>Latitude in decimal degrees (+ means North)</string>
                   </property>
                   <property name="text">
                    <string>Lat/Long:</string>
                   </property>
                   <property name="margin">
                    <number>5</number>
                   </property>
                   <property name="buddy">
                    <cstring>latlong_edit</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="LatLongEdit" name="latlong_edit">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="toolTip">
                    <string>Latitude in decimal degrees (+ means North)</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="timezoneHLayout">
                 <item>
                  <widget class="QLabel" name="timezoneLabel">
                   <property name="text">
                    <string>TZ:</string>
                   </property>
                   <property name="margin">
                    <number>5</number>
                   </property>
                   <property name="buddy">
                    <cstring>timezone</cstring>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="timezone">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="locationVerticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>503</width>
               <height>416</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QTextBrowser" name="vrataSummary">
                <property name="html">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;h1 style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:600; color:#008000;&quot;&gt;(unknown location)&lt;/span&gt;&lt;/h1&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;(unknown vrata) on YYYY-MM-DD&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;Pāraṇam: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_table">
         <attribute name="title">
          <string>Table</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayoutTabTable">
          <item>
           <widget class="HtmlBrowser" name="tableTextBrowser"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_daybyday">
         <attribute name="title">
          <string>Day-by-day</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayoutTabDaybyday">
          <item>
           <widget class="QTextBrowser" name="daybydayBrowser"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPa_c_gam">
    <property name="title">
     <string>&amp;Pañcāṅgam</string>
    </property>
    <addaction name="actionE_xit_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuPa_c_gam"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionE_xit_2">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HtmlBrowser</class>
   <extends>QTextBrowser</extends>
   <header>htmlbrowser.h</header>
  </customwidget>
  <customwidget>
   <class>LatLongEdit</class>
   <extends>QLineEdit</extends>
   <header>latlongedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
